terraform/modules/redis_prometheus_exporter/README.md

```diff
@@ -0,0 +1,23 @@
+# REDIS Exporter Module
+This is a terraform module used to install the REDIS-EXPORTER application into the Government PaaS, and setting some of the configuration.
```

Comment:
Please add the link to the redis exporter github

Reply:
added